provider/pkg/provider/provider.go

```diff
@@ -2552,6 +2552,9 @@ func (k *kubeProvider) tryServerSidePatch(oldInputs, newInputs *unstructured.Uns
 			return nil, nil, false
 		}
 	}
+	if err != nil {
+		return nil, nil, false
+	}
```

Comment:
Should we add an integration test?

Reply:
I worked on this for awhile today, but can't figure out a reasonable way to test with our existing framework since the bug is specific to using the CLI import command. Since the fix is just handling an unhandled error case, I'm going to defer testing for now.